packages/components/src/Glimmer/style/Sizes.css

```diff
@@ -0,0 +1,25 @@
+.small {
+  height: var(--space-small);
+}
+
+.base {
+  height: var(--space-base);
+}
+
+.large {
+  height: var(--space-large);
+}
+
+.larger {
+  height: var(--space-larger);
+}
+
+.largest {
+  height: var(--space-largest);
+}
+
+.auto {
```

Comment:
Just because I can already hear it being asked - why not all spacing sizes?

Reply:
and I would ask the question back, why should it?